samples/Chat/src/app/utils/uploadHandler.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* @conditional-compile-remove(attachment-upload) */
+import { AttachmentUploadHandler, AttachmentUploadOptions, AttachmentUploadManager } from '@azure/communication-react';
+/* @conditional-compile-remove(attachment-download) */
+import axios, { AxiosProgressEvent } from 'axios';
+/* @conditional-compile-remove(attachment-download) */
+import FormData from 'form-data';
+
+/* @conditional-compile-remove(attachment-download) */
+// max file size is 50MB
+const MAX_FILE_SIZE_MB = 50 * 1024 * 1024; // 50MB
+/* @conditional-compile-remove(attachment-download) */
+const UNSUPPORTED_FILES = ['exe', 'bat', 'dat'];
+/* @conditional-compile-remove(attachment-download) */
+const CONTAINER_NAME = '<CONTAINER_NAME>>';
```

Comment:
should this be an env variable?

Reply:
the container name cannot be an env variable since it's not shared for all endpoints ("/log" and "/file"). For example, the logs and files should not be in the same container and they should have their own container for easier maintenance 

<img width="602" alt="image" src="https://github.com/Azure/communication-ui-library/assets/109105353/5a02a50a-a100-4d0e-9eb0-afca4d90c3e6">

for file share, our container is either "uploads" (for file sharing in selfhost) or "acs-file-sharing-test" (for bug bash)

Reply:
talked with jim offline, updating this to actual container name